### PR TITLE
[Connectedk8s] Added proxy cert support in connectedk8s

### PR DESCRIPTION
**Pull request by Anumita**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by Anumita** [comment]
@akashkeshari can you please review?

**Review by akashkeshari — APPROVED** [review]

**Comment by yonzhan** [comment]
Connectedk8s

**Comment by Anumita** [comment]
@yungezz , can we get this reviewed and merged and released this week or on Monday latest? Review from connectedk8s side is done

**Comment by yungezz** [comment]
> @yungezz , can we get this reviewed and merged and released this week or on Monday latest? Review from connectedk8s side is done

sure.

**Review by yungezz — APPROVED** [review]
LGTM. Could you pls add test for this?

**Comment by Anumita** [comment]
@yungezz , I am still working out some issues while setting up aks with proxy (applying network policies in aks with proxy). Will add tests for proxy scenarios after i resolve them. Will raise that in a separate PR

**Comment by Anumita** [comment]
@yungezz , hey, any updates on this?

File: src/connectedk8s/azext_connectedk8s/_params.py [hunk 1]
```diff
@@ -21,6 +23,7 @@ def load_arguments(self, _):
         c.argument('https_proxy', options_list=['--proxy-https'], help='Https proxy URL to be used.')
         c.argument('http_proxy', options_list=['--proxy-http'], help='Http proxy URL to be used.')
         c.argument('no_proxy', options_list=['--proxy-skip-range'], help='List of URLs/CIDRs for which proxy should not to be used.')
+        c.argument('proxy_cert', options_list=['--proxy-cert'], help='Path to the certificate file for proxy')
```
Comment (fengzhou-msft) [inline src/connectedk8s/azext_connectedk8s/_params.py:26]: Why not add `type=file_type, completer=FilesCompleter()` here?

Comment (Anumita) [reply src/connectedk8s/azext_connectedk8s/_params.py]: Thanks for pointing it out. Have added it

**Pull request merged**